core/src/main/resources/org/elasticsearch/plugins/plugin-install.help

```diff
@@ -20,6 +20,8 @@ DESCRIPTION
 
     The notation of 'username/repository' refers to a github repository.
 
+    The argument can be an valid <url> which points to a download or file location for the plugin to be loaded from.
+
 EXAMPLES
 
     plugin install elasticsearch-analysis-kuromoji
```

Comment:
I think that we should rename our artifacts. I know it's late but if we want to be consistent we should remove elasticsearch- from all plugins